document-picture-in-picture/main.js

```diff
@@ -1,38 +1,34 @@
 
 const videoPlayer = document.getElementById("player");
 const playerContainer = document.getElementById("container");
-let togglePipButton;
-let pipActive = false;
-let pipWindow;
 
-const inPIPMessage = document.createElement("p");
-inPIPMessage.textContent = "Video player is currently in the separate Picture-in-Picture window.";
+const inPipMessage = document.querySelector(".in-pip-message");
+inPipMessage.style.display = "none";
```

Comment:
I would move that in css so that if JS is disabled, this text does not appear.

Reply:
Done in latest commit